pandas/core/series.py

```diff
@@ -3104,17 +3104,19 @@ def apply(self, func, convert_dtype=True, args=(), **kwds):
         """
         Invoke function on values of Series. Can be ufunc (a NumPy function
         that applies to the entire Series) or a Python function that only works
-        on single values
+        on single values.
 
         Parameters
         ----------
         func : function
-        convert_dtype : boolean, default True
+            Python function or NumPy ufunc.
+        convert_dtype : bool, default True
             Try to find better dtype for elementwise function results. If
-            False, leave as dtype=object
+            False, leave as dtype=object.
         args : tuple
-            Positional arguments to pass to function in addition to the value
-        Additional keyword arguments will be passed as keywords to the function
+            Positional arguments to pass to func in addition to the value.
+        **kwds : dict
```

Comment:
@datapythonista : I wonder if we should standardize and use `**kwargs` everywhere.

Reply:
In terms of documentation, the standard we defined was to always document `*args` and `**kwargs` if present, showing the stars in the name, but without the type (it'll always be tuple and dict).

Not sure if you mean using `kwargs` instead of `kwds`, or which standardization are you proposing here, but I'm +1 on being consistent everywhere.

I see that in this case `args` needs to be provided as a tuple (not as positional arguments) and `kwds` needs to be provided as keyword arguments (not as a dict). Not sure if there is a reason for this, but if it's not, I'd be in favor of changing it to be consistent (in a separate PR). @jreback ?

Reply:
@datapythonista : Indeed, I was referring to using `kwargs` across the board.  I'm also in favor of consistency in general.